python/_tskitmodule.c

```diff
@@ -6073,6 +6073,86 @@ TableCollection_get_file_uuid(TableCollection *self, void *closure)
     return Py_BuildValue("s", self->tables->file_uuid);
 }
 
+static PyObject *
+TableCollection_get_metadata(TableCollection *self, void *closure)
+{
+    PyObject *ret = NULL;
+
+    /* Py_BuildValue returns Py_None for zero length, we would rather
```

Comment:
We're doing this a few times, so we should make a function for it, i.e., ``get_metadata_bytes`` or something.

Reply:
Replaced with `PyBytes_FromStringAndSize`.

Reply:
Even better.